clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -237,11 +238,13 @@ public void removeListener(Listener listener) {
 
     private Cluster getClusterForCurrentTopics(Cluster cluster) {
         Collection<PartitionInfo> partitionInfos = new ArrayList<>();
+        List<Node> nodes = Cluster.empty().nodes();
```

Comment:
Yeah, you right. I realize that yet yesterday after pushing the repo (shrug). Gonna fix that.